src/app/filament/purge-filament/purge-filament.component.html

```diff
@@ -1,4 +1,4 @@
-<span class="filament-heading" i18n="@@purging-filament">purging filament</span>
+<span class="filament-heading" i18n="@@purging-filament">Check your filament color. If the color is right, click done.</span>
```

Comment:
purging is the official term and is shorter plus it fits better with the other screens ('loading filament', 'unloading filament', ...)